### DISCUSSION TITLE
Is it possible to disable the next button?

### DISCUSSION THREAD

**Discussion by ishaanbuildsthings** [gilbarbara/react-joyride#892] [Q&A] [answered]
I am using a controlled joyride, but I want the numbers to appear even for the steps where the user interacts with my UI. I've removed the 'next' text via `locale: { next: ''}`, but I also want the button to not be clickable. The reason I am trying to do this is it seems a lot easier than creating an entire custom component and matching the CSS to all my other steps. Is this possible?

Great library by the way, thank you!

**Comment by gilbarbara** [comment, marked as answer]
Hey @agrawalishaan 
No, only using a custom tooltip.
